src/cascadia/TerminalControl/IControlAppearance.idl

```diff
@@ -11,6 +11,8 @@ namespace Microsoft.Terminal.Control
         Windows.UI.Xaml.Media.Stretch BackgroundImageStretchMode;
         Windows.UI.Xaml.HorizontalAlignment BackgroundImageHorizontalAlignment;
         Windows.UI.Xaml.VerticalAlignment BackgroundImageVerticalAlignment;
+        Boolean IntenseIsBold;
+        // Boolean IntenseIsBright;

```

Comment:
Why is `IntenseStyle` declared in the `Microsoft.Terminal.Settings.Model` instead of the `Microsoft.Terminal.Control` namespace? I guess you need to convert the enum into two bools at some point, but it's just weird to see two super similar settings in one place imo.